src/Components/GenericExpressCheckout/CustomerRegistrationUtil.php

```diff
@@ -92,20 +108,16 @@ private function extractBillingData(array $response, string $key, string|null $a
             }
         }
 
-        if ($alternateKey === null
```

Comment:
just to make sure - did you test PayPal Express (v1) und Amazon Pay deeply, to make sure this removal will not break these integrations?

Reply:
Yes, I have tested both